### DISCUSSION TITLE
Filter verbinden

### DISCUSSION THREAD

**Discussion by Pieth13** [Schrolli91/BOSWatch#560] [General]
Hallo!
 
Ist es möglich mehrere Filter in der config.ini zu verbinden, zum Beispiel nur Nachrichten mit einem bestimmten Text und vom SubRIC 4 (d)?

Danke!

**Comment by Schrolli91** [comment]
Mit BOSWatch 3 geht das ohne Probleme - Die Architektur der Version 2 stößt da leider an ihre Grenzen.
https://github.com/BOSWatch/BW3-Core
